src/tabs.scss

```diff
@@ -179,6 +160,25 @@ $block: #{$fd-namespace}-tabs;
       }
     }
 
+    @include fd-selected() {
+      .#{$block}__tag {
+        color: var(--fdTabs_Selected_Color);
+      }
+
+      .#{$block}__icon {
+        color: var(--fdTabs_Selected_Icon_Color);
+        background-color: $fd-tabs-selected-color;
+      }
+
+      .#{$block}__icon,
+      .#{$block}__tag,
+      & {
+        &::after {
+          background-color: var(--fdTabs_Selected_Color);
+        }
+      }
+    }
+
```

Comment:
I think it will be better to mention about this change separately. I saw you added annotation, that many component has been changed, but in this case it requires serious action from user.